Modules/Invoices/Feature/Modules/InvoicesExportImportTest.php

```
@@ -16,252 +19,214 @@ class InvoicesExportImportTest extends AbstractCompanyPanelTestCase
 
     #[Test]
     #[Group('export')]
-    public function export_invoices_downloads_csv_with_correct_data(): void
+    public function it_dispatches_csv_export_job(): void
     {
         /* Arrange */
+        Queue::fake();
+        Storage::fake('local');
         $invoices = Invoice::factory()->for($this->company)->count(3)->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$invoices'. (undefined)

(UnusedLocalVariable)